### ISSUE TITLE
Error in resource "pfsense_dnsresolver_hostoverride" "examples" 

### ISSUE THREAD

**Issue by Pedroars22** [marshallford/terraform-provider-pfsense#15]
I have the following error

![image](https://github.com/marshallford/terraform-provider-pfsense/assets/64036816/e9d97001-b7d0-47bb-9a47-ef0142038595)

**Comment by Taegost** [comment]
I'm receiving the same error in my code as well

**Comment by marshallford** [comment]
Sorry about limited communication. What version of the provider are you using? Could you confirm that the provider source is correctly set in your `required_providers` block (example below)?

```
terraform {
  required_providers {
    pfsense = {
      source = "marshallford/pfsense"
      version = "<some version>"
    }
  }
}
```

**Comment by Taegost** [comment]
> Sorry about limited communication. What version of the provider are you using? Could you confirm that the provider source is correctly set in your `required_providers` block (example below)?
> 
> ```
> terraform {
>   required_providers {
>     pfsense = {
>       source = "marshallford/pfsense"
>       version = "<some version>"
>     }
>   }
> }
> ```

I was using the latest version, 0.7.0. Honestly, I dropped TF entirely, Ansible with PfSensible has way more functionality for managing pfSense and is much simpler.

**Comment by marshallford** [comment]
@Taegost Huh strange, I'll have to look into this error further. As for the move to Ansible -- fair enough, the initial focus of the provider has been just to support home network DNS configuration akin to the existing Terraform support for AWS Route 53 or Infoblox. Lastly, a quick plug: If you do find yourself missing Terraform while using PfSensible, checkout the other provider I maintain for running Ansible playbooks via Terraform: https://github.com/marshallford/terraform-provider-ansible. Thanks for the feedback/reply!

**Comment by aRustyDev** [comment]
I'm also having this same issue, not sure where to try poking to investigate further. Any thoughts?
Heres my current provider config, trying to implement the `pfsense_dnsresolver_hostoverride`
```hcl
terraform {
  required_providers {
    ...
    pfsense = {
      source = "marshallford/pfsense"
      version = "~> 0.7.0"
    }
  }
}

provider "pfsense" {
  url      = "https://pfsense.domain.tld"
  username = "user.name"
  password = var.pfsense_password
  tls_skip_verify = true
}
```

**Comment by pasquale1981** [comment]
it works up to 0.4.0 version.
From 0.5.0 on it shows that error.
(terraform v1.8.5)

**Comment by marshallford** [comment]
Hello all -- Thank you so much for your patience. I have published version `0.7.1` of the provider which upgrades the terraform plugin framework go library to resolve [this well known issue.](https://github.com/hashicorp/terraform-plugin-framework/issues/853) Please give it a go!
